src/SshNetTests/SshTests.cs

```diff
@@ -63,7 +63,7 @@ public void Ssh_ShellStream_Exit()
                         shellStream.ReadLine();
                         Assert.Fail();
                     }
-                    catch (NullReferenceException ex)
+                    catch (NullReferenceException)
```

Comment:
Non-related to this PR, but ```ShellStream.ReadLine()``` should not throw ```NullPointerException```, just return null. IMO this should be changed, but probably not in this PR.

Reply:
It should indeed, but not related to this PR.